lib/logstash/devutils/rspec/logstash_helpers.rb

```diff
@@ -67,8 +67,7 @@ def sample(sample_event, &block)
       end
 
       let(:results) do
-        pipeline.filters.each(&:register)
-
+        # Java pipeline (since 6.1) registers filters from start_workers
         pipeline.run_with(event)
```

Comment:
It is not clear to me where `TestPipeline#run_with` comes from, or how that ends up calling `Pipeline#start_workers` on its superclass that will do the requisite filter registration. Can you help me find this chain?